### ISSUE TITLE
请问是否有en-zh模型？

### ISSUE THREAD

**Issue by lidh15** [zwhe99/SelfTraining4UNMT#1]
为何在de fr之外选择了比较小的语种而没有zh

**Comment by zwhe99** [comment]
Guillaume Lample and Alexis Conneau. 2019. Cross-lingual language model pretraining. NeurIPS.

**Comment by zwhe99** [comment]
> 为何在de fr之外选择了比较小的语种而没有zh

@lidh15 你好。我们在语言选择上follow了以下两篇文章，并且使用了他们提供的预训练模型，所以暂时没有en-zh模型。

[1] Guillaume Lample and Alexis Conneau. 2019. Cross-lingual language model pretraining. NeurIPS.
[2] Kaitao Song, Xu Tan, Tao Qin, Jianfeng Lu, and Tie-Yan Liu. 2019. Mass: Masked sequence to sequence pre-training for language generation. In ICML.
